libjsonutils/cmake/FindRapidJSON.cmake

```diff
@@ -1,46 +1,55 @@
 #FindRapidJSON.cmake
-# 
+#
 # Finds the rapidjson library
 #
+# from http://rapidjson.org/
+#
 # This will define the following variables
 #
-#    RapidJSON_FOUND
-#    RapidJSON_INCLUDE_DIRS
+#    RAPIDJSON_FOUND
+#    RAPIDJSON_VERSION
+#    RAPIDJSON_INCLUDE_DIRS
 #
 # and the following imported targets
 #
 #     RapidJSON::RapidJSON
 #
 # Author: Pablo Arias - pabloariasal@gmail.com
+#
+# clausklein$ cmakelint *.cmake
+# FindRapidJSON.cmake:0: Find modules should use uppercase names;
+#   consider using FindRAPIDJSON.cmake [convention/filename]
```

Comment:
What convention is this? It sounds really weird to me.

Reply:
https://pypi.org/project/cmakelint/